libraries/app/application.cpp

```diff
@@ -334,29 +339,28 @@ class application_impl : public graphene::net::node_delegate
                 if (_options->count("replay-blockchain"))
                 {
                     ilog("Replaying blockchain on user request.");
-                    _chain_db->reindex(_data_dir / "blockchain", _shared_dir, _shared_file_size);
+                    _chain_db->reindex(_data_dir / "blockchain", _shared_dir, genesis_state, _shared_file_size);
```

Comment:
Not logical insert genesis between 2 parameters describing single entity

Reply:
agree, fixed.